chain/messagepool/messagepool.go

```diff
@@ -135,9 +135,9 @@ type MessagePool struct {
 	localAddrs map[address.Address]struct{}
 
 	// do NOT access this map directly, use getPendingMset, setPendingMset, deletePendingMset, forEachPending, and clearPending respectively
-	pending map[address.Address]*msgSet
+	pending sync.Map //threadsafe map[address.Address]*msgSet
```

Comment:
Is `*msgSet` immutable? Afaict it appears to be, but if it's not just this sync map won't be threadsafe.

Reply:
let's confirm this and/or create locks/safety for msgSet